src/AppBundle/Controller/SiteController.php

```diff
@@ -1,18 +1,53 @@
 <?php
-/**
- * A controller to deal with overall site stuff
- * @author David Christian Liedle <david.liedle@gmail.com>
- *
- */
+
 namespace AppBundle\Controller;
 
 use eZ\Bundle\EzPublishCoreBundle\Controller;
 use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
 
 class SiteController extends Controller
```

Comment:
I would prefer to see this as a service, especially without extending the controller.